src/guides/v2.3/extension-dev-guide/view-models.md

```diff
@@ -139,3 +139,5 @@ $postArray = $viewModel->getPostData(
     ['product' => $_item->getEntityId()]
 );
 ```
+{:.bs-callout-info}
+If you use `argument name="view_model"` you can access it using `$viewModel = $block->getViewModel();` Or You can use `argument name="viewModel"` and can access `$viewModel = $block->getData('viewModel');`
```

Comment:
IF this is true then the to changes above are mutually exclusive - only one of them should have been changed.

Reply:
yes, It depends on implementation! either we can use Getter/Setter OR we can get Data of argument!

Reply:
@BarnyShergold  First change I made as per example and second change is from actual changes from file [https://github.com/magento/magento2/blob/2.4-develop/app/code/Magento/Catalog/view/frontend/layout/catalog_product_view.xml#L178](url)